gradle.properties

```diff
@@ -55,6 +55,7 @@ reflectionsVersion = 0.9.12
 retrofitVersion = 2.9.0
 semverVersion = 3.1.0
 simpleExcelVersion = 1.1
+springCoreVersion = 5.2.8.RELEASE
```

Comment:
Commit message: "limitation**s**." Instead of the vague "several limitations", better mention the two specific limitations that we suffer from, which are no support for case-insensitive matching and the "**/" issue.

Reply:
Done.

Reply:
Commit message: s/AntPatchMatcher/AntPathMatcher/

I believe it's odd to introduce a dependency without actually using it. So how about simply squashing this with the next commit?

But, as stated before, I'm still not feeling 100% comfortable with introducing a dependency on `spring-core` just for a path matcher, even if `spring-core` has fewer (transitive) dependencies than I thought. The docs for [AntPathMatcher](https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/util/AntPathMatcher.html) say "Part of this mapping code has been kindly borrowed from Apache Ant", so just using Ant to get the Ant-style matcher seem much more straight forward to me.

Have you considered using Ant directly?

Reply:
Done.

Reply:
Here is another lib borrowing the code from spring: https://github.com/apache/shiro/blob/master/core/src/main/java/org/apache/shiro/util/AntPathMatcher.java.

Reply:
Regarding using Path matching code from Apache Ant, I haven't actually found the class for it.

Reply:
Anyhow, can we live with `spring-core` for now, and file a ticket to investigate getting rid of it - @sschuberth ?

Reply:
> I prefer to not squash the commits as I disagree that it's odd as is.

Just to explain myself further: To me this is like (temporarily) committing dead code, which is something that we also do not do. But I won't insist on squashing, and will not block this.

> How about keeping `spring-core` for now and I'll promise to do the research a bit later and switch to the `Ant` one in case it makes sense?

Good enough for me.